test/input_gen/genModelsRecurrent_v2.py

```diff
@@ -56,40 +56,39 @@ def forward(self, inputs, labels):
         return ret, loss
 
 class LSTMStacked(torch.nn.Module):
```

Comment:
I wonder why change this unit test than add another one?

Reply:
Previous unittest was test lstm layer works well when unrolling the lstm layer like lstmcell layer which is not applicable with this pr.